package.json

```diff
@@ -27,6 +27,7 @@
   "files": [
     "index.js",
     "src/**/*.js",
+    "scripts/*.js",
```

Comment:
i thought we were going to add a post-install hook to get that large dataset

Reply:
see above comment